BLUESPAWN-client/src/hunt/hunts/HuntT1060.cpp

```diff
@@ -63,12 +63,36 @@ namespace Hunts {
 		int detections = 0;
 		
 		for(auto& key : RunKeys){
-			for(auto& detection : CheckKeyValues(HKEY_LOCAL_MACHINE, key)){
+			for (auto& detection : CheckKeyValues(HKEY_LOCAL_MACHINE, key)) {
 				if (EvaluateFile(detection.ToString(), reaction)) {
 					reaction.RegistryKeyIdentified(std::make_shared<REGISTRY_DETECTION>(detection));
 				}
 				detections++;
 			}
+			for (auto& sub : CheckSubkeys(HKEY_LOCAL_MACHINE, key)) {
+				for (auto& detection : CheckKeyValues(HKEY_LOCAL_MACHINE, sub.GetNameWithoutHive())) {
```

Comment:
What's going on here? The whole purpose of CheckSubkeys is to check in different hives. Why would you use it and then swap the hive for HEKY_LOCAL_MACHINE?